fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayFragment.kt

```diff
@@ -194,6 +194,15 @@ class TabsTrayFragment : AppCompatDialogFragment() {
         return tabsTrayDialog
     }
 
+    override fun onPause() {
+        super.onPause()
+
+        // On some android versions (known about OneUI 2.5) entering animation is played every time
+        // DialogFragment goes though an onStart() lifecycle call. We want to prevent animation from
+        // being repeated after app returns from background.
+        dialog?.window?.setWindowAnimations(R.style.DialogFragmentRestoreAnimation)
```

Comment:
If it's a good approach (and I couldn't find a different one, to be honest), we might want to disable enter animation after onPause() on all of the dialog fragments, since they all have that problem on samsung android 10 (maybe others as well, but the reporter had that particular device and I happen to have it as well)